epistemix_platform/tests/test_job_results_endpoint.py

```
@@ -102,22 +104,69 @@ def setup_runs_with_urls(self):
-        """Test that the endpoint returns URLs for all runs with URLs for the specified job."""
-        response = client.get(
-            "/jobs/results?job_id=100",
-            headers={"Offline-Token": bearer_token, "Fredcli-Version": "1.0.0"},
-        )
+        """Test that the endpoint generates presigned URLs for all runs (batch operation)."""
+        # Mock the controller's get_run_results_download to return presigned URLs
+        with patch(
+            "epistemix_platform.app.get_job_controller"
+        ) as mock_get_controller:
+            from returns.result import Success
+            from unittest.mock import Mock
```

Flake8 (7.3.0)

[error] redefinition of unused 'Mock' from line 8

(F811)